src/navigation/PairHardwareWalletNavigator.tsx

```diff
@@ -57,10 +57,17 @@ export function PairHardwareWalletNavigator() {
     <BackgroundProvider color="surfaceSecondary">
       {({ backgroundColor }) => (
         <SimpleSheet backgroundColor={backgroundColor as string} onDismiss={onDismiss} scrollEnabled={false}>
+          {(currentRouteName === Routes.PAIR_HARDWARE_WALLET_INTRO_SHEET ||
+            currentRouteName === Routes.PAIR_HARDWARE_WALLET_SEARCH_SHEET) && (
```

Comment:
small nitpick but can you do a `.includes()` here instead?

Reply:
ts complained
and now I'm thinking do you think the legibility is worth it? I don't think it gets optimized out, so it creates a new array and then loops thru it, instead of two simple checks